AFMReader/jpk.py

```diff
@@ -105,17 +158,19 @@ def _get_z_scaling(tif: tifffile.tifffile, channel_idx: int, jpk_tags: dict[str,
             .name
         )
 
-        scaling = tif.pages[channel_idx].tags[scaling_name].value
-        offset = tif.pages[channel_idx].tags[offset_name].value
+        scaling = float(_get_tag_value(tif.pages[channel_idx], scaling_name))
+        offset = float(_get_tag_value(tif.pages[channel_idx], offset_name))
     elif scaling_type == "NullScaling":
-        scaling = 1
-        offset = 0
+        scaling = 1.0
+        offset = 0.0
     else:
         raise ValueError(f"Scaling type {scaling_type} is not 'NullScaling' or 'LinearScaling'")
     return scaling, offset
 
 
-def load_jpk(file_path: Path | str, channel: str, config_path: Path | str | None = None) -> tuple[np.ndarray, float]:
+def load_jpk(
+    file_path: Path | str, channel: str, config_path: Path | str | None = None, flip_image: bool = True
```

Comment:
Whilst it passes `mypy` as is, to be consistent with the `optional` type hint...

```suggestion
    file_path: Path | str, channel: str, config_path: Path | str | None = None, flip_image: bool | None = True
```